app/repositories/application/ReportingRepository.scala

```diff
@@ -94,6 +95,31 @@ class ReportingMongoRepository(timeZoneService: TimeZoneService)(implicit mongo:
     reportQueryWithProjectionsBSON[CandidateProgressReportItem](query, projection)
   }
 
+  override def applicationsForEdipReport(frameworkId: String): Future[List[ApplicationForEdipReport]] = {
+    val query = BSONDocument("$and" ->
+      BSONArray(
+        BSONDocument("frameworkId" -> frameworkId),
+        BSONDocument("applicationRoute" -> "Edip"),
+        BSONDocument(s"progress-status.${ProgressStatuses.PHASE1_TESTS_COMPLETED}" -> true),
```

Comment:
Is this right? The story states "GIVEN any EDIP candidate is in the PHASE_1_TESTS_COMPLETED state or beyond". How does this handle the 'beyond'?